src/stars.css

```diff
@@ -0,0 +1,12 @@
+.star {
+  width: 16px;
+  height: 16px;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.product-rating div:nth-child(-n+4) {
+  background-image: url("images/star-active.svg");
+}
```

Comment:
According to the task:
![image](https://user-images.githubusercontent.com/80700247/229275052-47e8e8f3-1acf-4b62-95dd-3e89caed66ef.png)